### ISSUE TITLE
Class properties: Useless documentation comment with @inheritDoc (false positive)

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1478]
```xml
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <rule ref="SlevomatCodingStandard.TypeHints.PropertyTypeHint"/>
    <rule ref="SlevomatCodingStandard.Commenting.UselessInheritDocComment"/>
</ruleset>
```

```php
<?php declare(strict_types = 1);

class Base {
    /**
     * @var array<string>
     */
    protected array $test = [];
}

class InheritDoc extends Base {
    /**
     * @inheritDoc
     */
    protected array $test = [];
}

class InheritDocAfterFix {
    protected array $test = [];
}
```

```
-----------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
-----------------------------------------------------------------------------------------------------------
 11 | ERROR | [x] Useless documentation comment with @inheritDoc.
 18 | ERROR | [ ] @var annotation of property \NoTypeHint::$test does not specify type hint for its items.
-----------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------
```

_v[8.7.0](https://github.com/slevomat/coding-standard/releases/tag/8.7.0)_

**Comment by kukulich** [comment]
See https://github.com/slevomat/coding-standard/issues/399

**Comment by LastDragon-ru** [comment]
In this case `@inheritDoc` needed to avoid copy pasting `@var array<string>` into all subclasses. Without it `@var annotation of property \NoTypeHint::$test does not specify type hint for its items.` will occur :(

Perhaps it is possible to disable "Forces to specify what's in traversable types like array, iterable and \Traversable." (that is already covered by phpstan in our case) for `SlevomatCodingStandard.TypeHints.PropertyTypeHint` somehow? (of course without `@phpcsSuppress`).
